website/docs/cloud-docs/api-docs/workspaces.mdx

```diff
@@ -485,7 +485,7 @@ Properties without a default value are required.
 | `data.attributes.vcs-repo.ingress-submodules` | boolean        | (previous value) | Whether submodules should be fetched when cloning the VCS repository.                                                                                                                                                                                                                                                                                                                       |
 | `data.attributes.vcs-repo.identifier`         | string         | (previous value) | A reference to your VCS repository in the format :org/:repo where :org and :repo refer to the organization and repository in your VCS provider. The format for Azure DevOps is `:org/:project/_git/:repo`.                                                                                                                                                                                  |
 | `data.attributes.working-directory`           | string         | (previous value) | A relative path that Terraform will execute within. This defaults to the root of your repository and is typically set to a subdirectory matching the environment when multiple environments exist within the same repository.                                                                                                                                                               |
-| `data.attributes.drift_detection`             | boolean | `false` | Whether Terraform Cloud will perform [drift detection assessments](/cloud-docs/workspaces/settings/drift-detection) for the workspace. Only available for Business tier organizations, in workspaces running Terraform version 0.15.4+ and operating in [Remote execution mode](/cloud-docs/workspaces/settings#execution-mode). |
+| `data.attributes.drift_detection`             | boolean | `false` | Whether Terraform Cloud will perform drift detection assessments for the workspace. Only available for Business tier organizations, in workspaces running Terraform version 0.15.4+ and operating in [Remote execution mode](/cloud-docs/workspaces/settings#execution-mode). Not available in Terraform Enterprise. |
```

Comment:
```suggestion
| `data.attributes.drift_detection`             | boolean | `false` | Whether or not Terraform Cloud performs drift detection assessments for the workspace. Only available for Business tier organizations, in workspaces running Terraform version 0.15.4+ and operating in [Remote execution mode](/cloud-docs/workspaces/settings#execution-mode). Not available in Terraform Enterprise. |
```

Suggest changing "Whether Terraform Cloud will perform drift detection" to "Whether or not Terraform Cloud performs drift detection" to avoid future tense.